interchange/DeviceResources.capnp

```diff
@@ -73,22 +73,23 @@ using TileTypeSiteTypeIdx = UInt32;
 
 struct Device {
 
-  name            @0 : Text;
-  strList         @1 : List(Text) $hashSet();
-  siteTypeList    @2 : List(SiteType);
-  tileTypeList    @3 : List(TileType);
-  tileList        @4 : List(Tile);
-  wires           @5 : List(Wire);
-  nodes           @6 : List(Node);
-  primLibs        @7 : Dir.Netlist; # Netlist libraries of Unisim primitives and macros
-  exceptionMap    @8 : List(PrimToMacroExpansion); # Prims to macros expand w/same name, except these
-  cellBelMap      @9 : List(CellBelMapping);
-  cellInversions @10 : List(CellInversion);
-  packages       @11 : List(Package);
-  constants      @12 : Constants;
-  constraints    @13 : Constraints;
-  lutDefinitions @14 : LutDefinitions;
-  parameterDefs  @15 : ParameterDefinitions;
+  name                  @0 : Text;
+  strList               @1 : List(Text) $hashSet();
+  siteTypeList          @2 : List(SiteType);
+  tileTypeList          @3 : List(TileType);
+  tileList              @4 : List(Tile);
+  wires                 @5 : List(Wire);
+  nodes                 @6 : List(Node);
+  primLibs              @7 : Dir.Netlist; # Netlist libraries of Unisim primitives and macros
+  exceptionMap          @8 : List(PrimToMacroExpansion); # Prims to macros expand w/same name, except these
+  cellBelMap            @9 : List(CellBelMapping);
+  cellInversions       @10 : List(CellInversion);
+  packages             @11 : List(Package);
+  constants            @12 : Constants;
+  constraints          @13 : Constraints;
+  lutDefinitions       @14 : LutDefinitions;
+  belChainsDefinitions @15 : BelChainsDefinitions;
```

Comment:
why is this before `parameterDefs`? afaik that will cause a backwards compatibility issue